ote_sdk/ote_sdk/entities/optimization_parameters.py

```diff
@@ -34,5 +33,5 @@ class OptimizationParameters:
     """
 
     resume: bool = False
-    update_progress: Callable[[int], None] = default_progress_callback
+    update_progress: UpdateProgressCallback = default_progress_callback
```

Comment:
what is this for?

Reply:
how will it work with  https://github.com/openvinotoolkit/mmdetection/blob/ote/mmdet/apis/ote/apis/detection/ote_utils.py#L113 ?

Do you need this change?

Reply:
@Ilya-Krylov Can you comment on why `default_progress_callback` for training with nncf differs from the callback for regular training? This difference causes such WAs. Also, this wa was only added to detection task, classification fails.

Reply:
It's because callbacks for training/inference/optimization have different types in the SDK. NNCF uses inference callback and pass to training, here is why there is a mismatch. There is no doubt that unifying callbacks in the SDK should be considered as a proper thing to do. But we should be sure, that these changes are breaking anything.

Reply:
So do you suggest trying to create a workaround for classification task or are there tests that can validate this functionality?

Reply:
Not quite get what's the matter with classification task. I'd propose to not change the SDK unless you have an agreement from all of the affected parties.